addons/l10n_in/data/account_tax_tag_data.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- GST Base tax tag -->
```

Comment:
Let's rather declare them in csv (waaay less lines, and the data are simple enough that we can afford that)